crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -366,7 +366,7 @@ fn excluded_only_compatible_version() {
           And because you require one of:
               package-a<2.0.0
               package-a>2.0.0
-          and you require package-b>=2.0.0,<3.0.0, we can conclude that the requirements are unsatisfiable.
+           and package-b>=2.0.0,<3.0.0, we can conclude that the requirements are unsatisfiable.
```

Comment:
This one I'm a little unsure about, but I think it's still clearer.

Reply:
Hm interesting thanks for flagging. It does seem like an improvement, I think.

Reply:
Just noticed the extra whitespace.

Reply:
I'm not sure that generalizes well for the "proof" that's being stated.